python/pyarrow/table.pxi

```diff
@@ -525,7 +525,8 @@ cdef class ChunkedArray(_PandasConvertible):
 
         return values
 
-    def __array__(self, dtype=None):
+    def __array__(self, dtype=None, copy=None):
+        # copy keyword can be ignored because to_numpy() already returns a copy
```

Comment:
Well, `copy=False` should then raise an error, no?

Reply:
Ah yes, that comment is from before I decided to handle the copy=False case for Array. Indeed we should just raise an error here that a no-copy is not possible.

Updated.